### PR TITLE
W-10978292 Exchange add ruleset asset jh

### PR DESCRIPTION
**Pull request by JennyHajee**
Added ruleset asset to Asset Types and Create an asset

### CODE REVIEW

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 1]
```diff
@@ -104,18 +104,21 @@ To create an API asset:
 * REST API - RAML: Provide a RAML API specification file. RAML specifications must be a RAML file (.raml).
 * REST API - OAS: Provide an OAS API specification file. OAS specifications can be either a YAML (.yaml) or JSON (.json) file. Exchange supports OAS 2.0 and OAS 3.0 specifications.
 * SOAP API - WSDL: Provide a WSDL API specification file. SOAP specifications file can be either a WSDL (.wsdl) or XML (.xml) file.
-* API Spec Fragment - RAML : Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml).
-* HTTP API - This asset does not require a file. This type of asset provides an API endpoint that is defined by API Manager.
+* AsyncAPI: Provide an AsyncAPI specification file in a YAML (.yaml) file.
+* API Spec Fragment - RAML: Provide an API Fragment RAML specification file. Fragment specifications must be a RAML file (.raml).
+* HTTP API: This asset does not require a file. This type of asset provides an API endpoint that is defined by API Manager.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
* HTTP API: This asset does not require a file. This asset type provides an API endpoint that is defined by API Manager.
```

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 2]
```diff
@@ -104,18 +104,21 @@ To create an API asset:
...
 .. Select *Choose File* to locate the API specification file.
 .. Select the main file of the API.
 +
 If the file is a ZIP, the selected main file must be in the root directory of the ZIP file. If the file is not a ZIP or if the file is a ZIP file with only one main file, then the main file is selected automatically.
 +
-. (Optional) Select *Advanced* and edit the advanced settings: *GroupId*, *AssetId*, *Version*, and *API version*. Exchange generates the group ID, asset ID, and version for you, and you can change these values here. You can change an API asset's version (asset version) and API version separately. The advanced settings are most often used to change the asset version.
-. (Optional) Select an option for *Lifecycle state*. The default is *Stable*.
+. To edit *GroupId*, *AssetId*, *Version*, and *API version*, click *Advanced*.
++
+ Exchange generates the group ID, asset ID, and version for you, and you can change these values here. You can change an API asset's version (asset version) and API version separately. The advanced settings are most often used to change the asset version.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
 Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. You can change an API's asset version and version separately. Use the advanced settings to change the asset version.
```

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 3]
```diff
@@ -144,7 +147,7 @@ Warnings are also shown when publishing an API Group to your public portal if al
 
 A custom asset lets you share information about any aspect of your organization such as announcements, documentation, videos, and sharing files. You can add an optional file to your Custom asset that users can download. The file is stored in Exchange.
 
-*Note:* Exchange only permits the following file types as the optional file in a Custom asset:
+Exchange only permits the following file types as the optional file in a Custom asset:
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
Exchange only permits the following file types as the optional file in a custom asset:
```

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 4]
```diff
@@ -162,11 +165,25 @@ To create a custom asset:
 . In Exchange, select *Publish new asset*.
 . Enter the asset portal name.
 . Select the asset type *Custom* from the drop-down list.
-. (Optional) To share a file with users, choose the file.
-. (Optional) Select *Advanced* and edit the advanced settings: *GroupId*, *AssetId*, and *Version*. Exchange generates the group ID, asset ID, and version for you, and you can change these values here. The advanced settings are most often used to change the asset version.
-. (Optional) Select an option for *Lifecycle state*. The default is *Stable*.
+. Click *Choose the file* to select the file to upload.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
. To select the file to upload, click *Choose the file*.
```
this keeps it parallel with how you worded the next step

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 5]
```diff
@@ -162,11 +165,25 @@ To create a custom asset:
...
+. To edit the *GroupId*, *AssetId*, and *Version*, click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values here. The advanced settings are most often used to change the asset version.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
```

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 6]
```diff
@@ -162,11 +165,25 @@ To create a custom asset:
...
+. Select an option for *Lifecycle state*. The default is *Stable*.
 . Select *Publish*.
 
+== Create a Ruleset Asset
+Rulesets are a group of rules or guidelines that define an application, for example best practices or industry-specific standards. Add a zip file with multiple YAML files that describe the new ruleset.
+
+. In Exchange, select *Publish new asset*.
+. Enter the asset portal name.
+. Select the asset type *Ruleset* from the drop-down list.
+. Click *Choose the file* to select the file to upload.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
. To select the file to upload, click *Choose the file* .
```

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 7]
```diff
@@ -162,11 +165,25 @@ To create a custom asset:
...
+Exchange generates the group ID, asset ID, and version for you, and you can change these values here. The advanced settings are most often used to change the asset version.
+. Select an option for *Lifecycle state*. The default is *Stable*.
 . Select *Publish*.
 
+== Create a Ruleset Asset
+Rulesets are a group of rules or guidelines that define an application, for example best practices or industry-specific standards. Add a zip file with multiple YAML files that describe the new ruleset.
+
+. In Exchange, select *Publish new asset*.
+. Enter the asset portal name.
+. Select the asset type *Ruleset* from the drop-down list.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
. From the drop-down list, select the asset type *Ruleset* .
```
This is just my particular thing 🤣  --I'm always going to put the preposition/navigation first

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 8]
```diff
@@ -162,11 +165,25 @@ To create a custom asset:
...
+. Click *Choose the file* to select the file to upload.
+. To edit the advanced settings: *GroupId*, *AssetId*, and *Version*, click *Advanced*.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
. To edit the advanced settings, *GroupId*, *AssetId*, and *Version*, click *Advanced*. 
```

File: modules/ROOT/pages/to-create-an-asset.adoc [hunk 9]
```diff
@@ -162,11 +165,25 @@ To create a custom asset:
...
+== Create a Ruleset Asset
+Rulesets are a group of rules or guidelines that define an application, for example best practices or industry-specific standards. Add a zip file with multiple YAML files that describe the new ruleset.
+
+. In Exchange, select *Publish new asset*.
+. Enter the asset portal name.
+. Select the asset type *Ruleset* from the drop-down list.
+. Click *Choose the file* to select the file to upload.
+. To edit the advanced settings: *GroupId*, *AssetId*, and *Version*, click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values here. The advanced settings are most often used to change the asset version.
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/to-create-an-asset.adoc]:
```suggestion
Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
```

**Review by IsaacEldridge — DISMISSED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**